src/components/SearchModal/CurrencyList.tsx

```diff
@@ -175,14 +176,18 @@ export function CurrencyRow({
       <Column style={{ alignItems: 'flex-end', gap: 2 }}>
         <RowFixed style={{ justifySelf: 'flex-end', gap: 15 }}>
           {showImported ? (
-            <DeleteButton onClick={onClickRemove} />
+            <DeleteButton onClick={onClickRemove} className="button-remove-import-token" />
```

Comment:
Please use data-testid instead of classname. Classname will be used to styling component, for testing, you can use data-testid attribute

Reply:
data-testid or data-cy-* 
https://docs.cypress.io/guides/references/best-practices

Reply:
I've updated. Please review again.